app/ts/library/humanEthersErrors.ts

```diff
@@ -0,0 +1,102 @@
+			case 'UNSUPPORTED_OPERATION':
+				return { warning: true, message: `Attempted to execute an unsupported operation: "${error.info ? JSON.stringify(error.info) : ''}". This is a bug and you should report it.` };
+			case 'SERVER_ERROR':
+				return { warning: true, message: `Could not communicate with server. ${error.info ? JSON.stringify(error.info) : ''}` };
+			case 'TIMEOUT':
+				return { warning: true, message: `Timeout during action "${error.operation}". ${error.reason}` };
+			case 'BAD_DATA':
+				return { warning: true, message: `EthersJS tried failed to understand this value: ${JSON.stringify(error.value)}. This is likely a bug and you should report it.` };
+			case 'CANCELLED':
+				return { warning: false, message: `Request was canceled by app. ${error.info ? JSON.stringify(error.info) : ''}` };
+			// Operational Errors
+			case 'BUFFER_OVERRUN':
+				return { warning: true, message: `Buffer overrun. This likely a bug and you should report it. ${error.info ? JSON.stringify(error.info): ''}` };
+			case 'NUMERIC_FAULT':
+				return { warning: true, message: `Failed to ${error.operation}. ${error.fault} with value ${error.value}` };
+			// Argument Errors
+			case 'INVALID_ARGUMENT':
+				return { warning: true, message: `EthersJS received an invalid argument, "${error.argument}" with value ${JSON.stringify(error.value)}. This is a bug and you should report it. ${error.stack ?? ''}` };
+			case 'MISSING_ARGUMENT':
+				return { warning: true, message: `EthersJS expected ${error.count} arguments and received ${error.expectedCount}. This is a bug and you should report it. ${error.stack ?? ''}` };
+			case 'UNEXPECTED_ARGUMENT':
+				return { warning: true, message: `EthersJS received too many arguments. This is a bug and you should report it. ${error.stack ?? ''}` };
+			// Blockchain Errors
+			case 'CALL_EXCEPTION':
+				if (error.receipt) {
+					return { warning: true, message: `Transaction was included in block #${error.receipt.blockNumber} but reverted${error.reason ? ` with error: ${error.reason}`: ''}` }
+				} else {
+					return { warning: true, message: error.reason ? `Transaction will fail. Call exeception during ${error.action}: ${error.reason}`: `The transaction will revert. ${error.reason ?? ''}` };
+				}
+			case 'INSUFFICIENT_FUNDS':
+				return { warning: true, message: `Account ${error.transaction.from} does not have enough funds for this transaction.` };
+			case 'NONCE_EXPIRED':
+				return { warning: true, message: `Account ${error.transaction.from} has already made a transaction with the same nonce. Call the transaction again with a new nonce.` }; // Whats human readable / user friendly here?
+			case 'REPLACEMENT_UNDERPRICED':
+				return { warning: true, message: `Unimplemented ${error.code}` };
+			case 'TRANSACTION_REPLACED':
+				return { warning: true, message: `Unimplemented ${error.code}` };
```

Comment:
Not sure for how to word these scenarios in a way that is retail user friendly.

Reply:
"The replacement transaction is underpriced"
"The transaction got replaced by a transaction with the same nonce"